test/list.go

```diff
@@ -422,12 +414,14 @@ func DescribeListTestsFor(ctx *common.TestContext, t TestCase) bool {
 
 			Context("Paging", func() {
 				Context("with max items query", func() {
-					It("returns smaller pages", func() {
+					It("returns smaller pages token and Link header", func() {
```

Comment:
The way "Paging" tests context is added will not test a combination of fieldqueries + labelQueries + paging. Not sure if its a big deal but if for example we introduce a bug that causes having a labelquery and paging query (token/max_items) to return a wrong result or error, we wont catch that with these tests

Reply:
The original tests are still using paging and the result is aggregated by the List function, so if something is broken other test Entries will cover it, not the tests in the Paging context.